tests/python/unittest/test_gluon_rnn.py

```diff
@@ -152,190 +124,7 @@ def test_lstmp():
         check_rnn_states(fused_states, stack_states, num_layers, True)
 
 
-@assert_raises_cudnn_not_satisfied(min_version='5.1.10')
-def test_lstm_cpu_inference():
-    # should behave the same as lstm cell
-    EXPECTED_LSTM_OUTPUT = np.array([[[0.72045636, 0.72045636, 0.95215213, 0.95215213],
-                                      [0.72045636, 0.72045636, 0.95215213, 0.95215213]],
-                                     [[0.95215213, 0.95215213, 0.72045636, 0.72045636],
-                                      [0.95215213, 0.95215213, 0.72045636, 0.72045636]]])
-    x = mx.nd.ones(shape=(2, 2, 2))
-    model = mx.gluon.rnn.LSTM(2, num_layers=6, bidirectional=True)
-    model.initialize(mx.init.One())
-
-    y = model(x).asnumpy()
-    mx.test_utils.assert_almost_equal(y, EXPECTED_LSTM_OUTPUT,
-                                      rtol=1e-3, atol=1e-5)
-
-
-def test_gru():
```

Comment:
Should this test be adapted to testing the front-end infer_shape functions instead?